spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/partitioner/dht/Token.scala

```diff
@@ -1,17 +1,61 @@
 package com.datastax.spark.connector.rdd.partitioner.dht
 
+import com.datastax.spark.connector.rdd.partitioner.MonotonicBucketing
+import com.datastax.spark.connector.rdd.partitioner.MonotonicBucketing.LongBucketing
+
 trait Token[T] extends Ordered[Token[T]] {
+  def ord: Ordering[T]
```

Comment:
This is needed because we can't compare both BigInt and Long's with ordered :( Our workaround was to pass the ordering for each type.